app/src/test/java/org/simple/clinic/editpatient/PatientEditScreenControllerTest.kt

```diff
@@ -101,15 +96,6 @@ class PatientEditScreenControllerTest {
     val address = PatientMocker.address()
     val phoneNumber: PatientPhoneNumber? = null
 
-    whenever(patientRepository.patient(patient.uuid)).thenReturn(Observable.just(patient.toOptional()))
-    whenever(patientRepository.address(address.uuid)).thenReturn(Observable.just(address.toOptional()))
```

Comment:
We can remove these stubs from `PatientEditScreenValidationUsingMockDateValidator` and `PatientEditScreenValidationUsingMockNumberValidatorTest` as well.

Reply:
That is coming in as part of the next PR.